### PR TITLE
Issue703

### PR DESCRIPTION
**Pull request by mreboud**
@dvandyk Can you have a look at the description and tell me what I should had? I think that adding a notebook that reproduces the B -> K ll predictions is necessary if we want to make this clear...

Github: Solves #703

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -123,3 +123,40 @@ If you installed via ``pip``
...
+   * - ``BFS2004``
+     - Next-to-leading order calculation based on the framework of QCD factorization and valid at small dilepton masses.
+     - [BFS:2001A], [BFS:2004A]
+
+   * - ``GP2004``
+     - Calculation of the non-local contributions valid for large dilepton masses.
+     - [GP:2004A]
+
+   * - ``GvDV2020``
+     - Parametrization based on unitarity and analyticity valid for small dilepton masses.
```
Comment (dvandyk) [inline doc/faq.rst:161]: Maybe add ``below the $\psi(2S)$ state``.

File: doc/faq.rst [hunk 2]
```diff
@@ -123,3 +123,40 @@ If you installed via ``pip``
...
+   * - ``Naive``
+     - Leading-order calculation.
+     -
+
+   * - ``BFS2004``
+     - Next-to-leading order calculation based on the framework of QCD factorization and valid at small dilepton masses.
+     - [BFS:2001A], [BFS:2004A]
+
+   * - ``GP2004``
+     - Calculation of the non-local contributions valid for large dilepton masses.
```
Comment (dvandyk) [inline doc/faq.rst:157]: Maybe: ``Calculation of the non-local contributions in a local OPE. This approach is expected to yield sensible results only for $q^2$-integrated observables above the open-charm threshold.``

File: doc/faq.rst [hunk 3]
```diff
@@ -123,3 +123,40 @@ If you installed via ``pip``
...
+   * - ``tag``
+     - Description
+     - Reference
+
+   * - ``Naive``
+     - Leading-order calculation.
+     -
+
+   * - ``BFS2004``
+     - Next-to-leading order calculation based on the framework of QCD factorization and valid at small dilepton masses.
```
Comment (dvandyk) [inline doc/faq.rst:153]: Maybe: ``Calculation of the non-local contribution based on the framework of QCD factorization and a perturbative treatment of the intermediate quark states. This approach is expected to yield sensible results only for $q^2 \ll 4 m_c^2$.``

File: doc/faq.rst [hunk 4]
```diff
@@ -123,3 +123,40 @@ If you installed via ``pip``
...
+.. list-table::
+   :widths: 15 55 30
+   :header-rows: 1
+
+   * - ``tag``
+     - Description
+     - Reference
+
+   * - ``Naive``
+     - Leading-order calculation.
```
Comment (dvandyk) [inline doc/faq.rst:149]: Hm, I disagree. ``Absence of any non-local contributions.``?

**Review by dvandyk — COMMENTED** [review]
Thanks @mreboud. I left some comments.

File: doc/faq.rst (hunk 4 above)
Comment (mreboud) [reply doc/faq.rst:149]: Not really, I think we are still using `C9eff` and `C7eff` at leading order in alpha_s, see e.g. https://github.com/eos/eos/blob/f3c073bf2bb9a2aa2af7740ab4a905a68c2c90b8/eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc#L88-L89

Comment (dvandyk) [reply doc/faq.rst:149]: The key is in the name of the class: ``c7eff`` and ``c9eff`` are the short-distance quantities, basically ``c7`` and ``c9`` modified to be invariant under the RGE to higher order in $\alpha_s$. As such, they are essential to complete the  local contributions but do not contain any non-local terms (despite arising from four-quark operators).

Comment (mreboud) [reply doc/faq.rst:149]: Ah yes sorry I am an idiot

**Review by dvandyk — APPROVED** [review]

**Pull request merged**